src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -702,6 +717,54 @@ export default {
 			this.$emit('update:isReactionsMenuOpen', true)
 		},
 
+		async forwardToNote() {
+			const NoteToSelf = this.$store.getters.conversationsList.filter(conversation => conversation.type === CONVERSATION.TYPE.NOTE_TO_SELF)[0]
+			const messageToBeForwarded = cloneDeep(this.messageObject)
```

Comment:
Is this the same code as in ForwardMessage.vue?

Reply:
Yes with toast messages addition. For now, the forwarding to note to self is omitting messages with type `{file}` like ForwardMessage.vue . If we want to also be able to forward pictures , files ... either : 
- Post the file link in self to note.
- Process and post it as in NewMessage but change the token to note to self's.

Otherwise, we can move the function to the messageStore to reduce redundancy.

Reply:
For files sharing, it will be touched anyway by https://github.com/nextcloud/spreed/pull/8949, so we can leave it for now
For method, let's reuse it with messagesStore. Toasts could be posted from component, just throw error from store action and add try-catch in Forwarder and MessageButtonsBar